integration-tests/Cargo.toml

```diff
@@ -20,14 +20,14 @@ pallet-circuit-breaker = { path = "../pallets/circuit-breaker",default-features
 # HydraDX dependencies
 pallet-omnipool-liquidity-mining = { path = "../pallets/omnipool-liquidity-mining", default-features = true}
 
-pallet-asset-registry = { git = "https://github.com/galacticcouncil/warehouse", rev = "a6085a6e0b435607403e0ff890fc61fab796fb46", default-features = false }
```

Comment:
> @enthusiastmartin why is the warehouse version change necessary?

similar to what i replied in the warehouse. We need to have everything at the same, otherwise you start having compilation error because various types and traits would be different in case of different warehouse versions.

it might be ok here, but sooner or later, something fails to compile because of that. so it is always better to set the same version everywhere.

monorepo ftw.